release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -148,6 +148,90 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
 [id="ocp-4-13-storage"]
 === Storage
 
+[id="ocp-4-13-storage-csi-vol-health"]
+==== CSI volume health (Technology Preview)
+Container Storage Interface (CSI) volume health is currently enabled, and supported as a Technology Preview feature. However, {product-title} currently does not ship any CSI drivers that support this feature.
+
+[id="ocp-4-13-storage-csi-vol-detach-non-graceful-shutdown"]
+==== CSI driver volume detach with non-graceful node shutdown (Technology Preview)
+Container Storage Interface (CSI) drivers can now automatically detach volumes when a node goes down non-gracefully. When a non-graceful node shutdown occurs, you can then manually add an out-of-service taint on the node to allow volumes to automatically detach from the node. This feature is supported with Technology Preview status.
+
+For more information, see ???.
+
+[id="ocp-4-13-storage-vsphere-encryption-support"]
+==== VMware vSphere encryption support is generally available
+You can encrypt virtual machines (VMs) and persistent volumes (PVs) on {product-title} running on vSphere. This feature allows user-provisioned persistent storage to be encrypted over wire.
+
+For more information, see ???.
+
+[id="ocp-4-13-storage-vpshere-top-support-multi-datacenters"]
+==== VMware vSphere CSI topology support for multiple datacenters is generally available
+{product-title} 4.12 introduced the ability to deploy {product-title} for vSphere on different zones and regions, which allows you to deploy over multiple compute clusters, thus helping to avoid a single point of failure. {product-title} 4.13 introduces support for deploying over multiple datacenters also.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-top-aware_persistent-storage-csi-vsphere[vSphere CSI topology].
+
+[id="ocp-4-13-storage-multiple-default-sc"]
+==== Creating more than one default storage class is generally available
+{product-title} 4.13 allows you create more than one default storage class. This feature makes it easier to change the default storage class because you can create a second storage class defined as the default. You then temporarily have two default storage classes before removing default status from the previous default storage class. While it is acceptable to have multiple default storage classes for a short time, you should ensure that eventually only one default storage class exists.
+
+For more information, see ???.
+
+[id="ocp-4-13-storage-managing-default-sc"]
+==== Managing the default storage class is generally available 
+{product-title} 4.13 introduces the `spec.storageClassState` field in the `ClusterCSIDriver` object, which allows you to manage the default storage class to accomplish several different objectives:
+
+* Enforcing static provisioning by disabling dynamic provisioning.
```

Comment:
I feel this is not a good one to introduce this feature, wdyt? @ropatil010 @jsafrane

Reply:
Limit storage consumption to static provisioning?

Reply:
@gcharot @duanwei33 Do you want to change the wording of the first bullet point to "Limit storage consumption to static provisioning" and move it to be the last bullet point?

Reply:
Yes, I agree move it to the last. Thanks!

Reply:
Done.